src/main/java/fi/metatavu/metaform/server/persistence/model/AuditLogEntry.java

```diff
@@ -0,0 +1,192 @@
+package fi.metatavu.metaform.server.persistence.model;
+
+import fi.metatavu.metaform.client.model.AuditLogEntryType;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+/**
+ * JPA entity representing audit log entry
+ *
+ * @author Katja Danilova
+ */
+@Entity
+@Cacheable(true)
+@Cache(usage = CacheConcurrencyStrategy.TRANSACTIONAL)
+public class AuditLogEntry {
+
+  @Id
+  private UUID id;
+
+  @Column(nullable = false)
+  @NotNull
+  private UUID userId;
+
+  @Column(nullable = false)
+  @NotNull
+  private OffsetDateTime time;
```

Comment:
This should be refactored to createdAt for consistency's sake and translated into automatically propaged field (see https://github.com/Metatavu/metaform-api/blob/9c216adaf07591ec47c800fc15c2cd938189eee7/src/main/java/fi/metatavu/metaform/server/persistence/model/Reply.java#L119)